auslib/admin/views/rules.py

```diff
@@ -298,14 +299,21 @@ def __init__(self):
 
     @requirelogin
     def _post(self, transaction, changed_by):
-        if request.json and request.json.get("data_version"):
-            form = ScheduledChangeExistingRuleForm()
-        else:
-            form = ScheduledChangeNewRuleForm()
+        if request.json.get("change_type") != "delete":
```

Comment:
I think change_type need to be a required parameter. The way this is written now, if it's not specified we assume it's a Scheduled Deletion, which seems like a bad assumption to make. You should probably adjust the Form to give it a Required() Validator. This should be easier if change_type is only present in the Forms for new Scheduled Changes.

And a small nit: It would be a bit easier to read this if you used == here, and put update/insert in the else block.